python/semantic_kernel/planners/function_calling_stepwise_planner/utils.py

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+
+def read_file(file_path: str) -> str:
+    """Read the given configuration or file
+    related to the FunctionCallingStepwisePlanner"""
+    with open(file_path, "r") as file:
```

Comment:
Is this function really necessary? Why not a simple `open(file_path, "r").read()` where reading a file is needed? Seems like replacing a single line statement for a single line statement, no?

Reply:
Sure, I can make that update. Thanks!